### PR TITLE
HSEARCH-4913 Generate module-info for `hibernate-search-mapper-orm-batch-jsr352-jberet`

### PR DESCRIPTION
**Pull request by marko-bekhta**
https://hibernate.atlassian.net/browse/HSEARCH-4913

### CODE REVIEW

File: build/parents/public/pom.xml [hunk 1]
```diff
@@ -231,7 +230,12 @@
                                 </moduleInfo>
                             </module>
                             <jdepsExtraArgs>
-                                <arg>--multi-release=9</arg>
+                                <!--
+                                    Running jdpes tool will fail without this option, if there are any multi-release jars
```
Comment (yrodiere) [inline build/parents/public/pom.xml:234]:
```suggestion
                                    Running the jdeps tool will fail without this option, if there are any multi-release jars
```

File: build/parents/public/pom.xml [hunk 2]
```diff
@@ -231,7 +230,12 @@
...
+                                    on the path. And since some modules might have such jars, e.g. util-common has Jandex,
+                                    it's safer to add this additional option:
```
Comment (yrodiere) [inline build/parents/public/pom.xml:236]: Okay. Does this mean we generate a multi-release JAR, or just that we pass an extra argument to `jdeps` during the build, with no impact on the produced JAR?

**Review by yrodiere — APPROVED** [review]
Works for me, thanks, though there's a typo :)

File: build/parents/public/pom.xml (hunk 2 above)
Comment (marko-bekhta) [reply build/parents/public/pom.xml:236]: From what it looks: only an extra argument. I've looked inside the resulting jar - module-info is just in the root, and there's no additional versions folder for multi-release

**Pull request merged**